plugins/node.d/http_loadtime

```diff
@@ -86,9 +85,8 @@ if [ "$1" = "config" ]; then
     do
         uri_short=`echo ${uri:0:30}`
         if [ "$uri_short" != "$uri" ]; then uri_short=$uri_short"..."; fi
-        esc_uri="$(clean_fieldname "$uri")"
-        echo $esc_uri".label $uri_short"
-        echo $esc_uri".info page load time"
+        echo `escapeUri $uri`".label $uri_short"
```

Comment:
I also quite like the `$( ... )` syntax more. Specially if you use twice the same var.